lib/rubocop/cop/rspec/leaky_local_variable.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks for outside local variables used in examples.
```

Comment:
I can’t recall if I’ve ever heard of a term “outside” applied to variables in Ruby. Should it be just “local”, or is it something more specific?
I apologise in advance if it’s something common; I’ve never heard of join_forces before this PR, too.

Reply:
Much better, thank you!